src/components/layouts/llama-vertical-scroll-layout.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2022 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { LitElement, html, TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+
+import { LlamaStorage } from '../../storage';
+
+@customElement('llama-vertical-scroll-layout')
+/**
+ * Default Llaminator layout - renders all items at once, vertically in a single column, and lets
+ * the browser handle scrolling in the normal way. Implements ../../llaminator.Layout.
+ */
+export class LlamaVerticalScrollLayout extends LitElement {
+  /**
+   * This component needs an open shadow root because LlamaItem depends on it.
+   *
+   * @return {LlamaVerticalScrollLayout} The current instance.
+   */
+  createRenderRoot() { return this; }
+  items: TemplateResult[] = [];
```

Comment:
nit: data members before methods? (optional I guess, not currently part of the style we're enforcing)

Reply:
done